code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -54,189 +67,97 @@
-		user << "<span class='warning'>Our sting appears ineffective against its DNA.</span>"
-		return 0
-	return 1
-
-/obj/effect/proc_holder/changeling/sting/false_armblade/sting_action(mob/user, mob/target)
-	add_logs(user, target, "stung", object="falso armblade sting")
-
-	if(!target.drop_item())
-		user << "<span class='warning'>The [target.get_active_hand()] is stuck to their hand, you cannot grow a false armblade over it!</span>"
-		return
-
-	if(ismonkey(target))
-		user << "<span class='notice'>Our genes cry out as we sting [target.name]!</span>"
-
-	var/obj/item/weapon/melee/arm_blade/false/blade = new(target,1)
-	target.put_in_hands(blade)
-	target.visible_message("<span class='warning'>A grotesque blade forms around [target.name]\'s arm!</span>", "<span class='userdanger'>Your arm twists and mutates, transforming into a horrific monstrosity!</span>", "<span class='italics'>You hear organic matter ripping and tearing!</span>")
-	playsound(target, 'sound/effects/blobattack.ogg', 30, 1)
-
-	spawn(600)
-		playsound(target, 'sound/effects/blobattack.ogg', 30, 1)
-		target.visible_message("<span class='warning'>With a sickening crunch, [target] reforms their [blade.name] into an arm!</span>", "<span class='warning'>[blade] reforms back to normal.</span>", "<span class='italics>You hear organic matter ripping and tearing!</span>")
-		qdel(blade)
-		user.update_inv_l_hand()
-		user.update_inv_r_hand()
-
-	feedback_add_details("changeling_powers","AS")
-	return 1
-
-
-/obj/effect/proc_holder/changeling/sting/extract_dna
-	name = "Extract DNA Sting"
-	desc = "We stealthily sting a target and extract their DNA."
-	helptext = "Will give you the DNA of your target, allowing you to transform into them."
-	sting_icon = "sting_extract"
-	chemical_cost = 25
-	dna_cost = 0
-
-/obj/effect/proc_holder/changeling/sting/extract_dna/can_sting(mob/user, mob/target)
-	if(..())
-		return user.mind.changeling.can_absorb_dna(user, target)
-
-/obj/effect/proc_holder/changeling/sting/extract_dna/sting_action(mob/user, mob/living/carbon/human/target)
-	add_logs(user, target, "stung", "extraction sting")
-	if(!(user.mind.changeling.has_dna(target.dna)))
-		user.mind.changeling.add_new_profile(target, user)
-	feedback_add_details("changeling_powers","ED")
-	return 1
-
 /obj/effect/proc_holder/changeling/sting/mute
 	name = "Mute Sting"
 	desc = "We silently sting a human, completely silencing them for a short time."
-	helptext = "Does not provide a warning to the victim that they have been stung, until they try to speak and cannot."
+	helptext = "Our target will not be alerted to their silence until they attempt to speak and cannot."
 	sting_icon = "sting_mute"
 	chemical_cost = 20
-	dna_cost = 2
+	dna_cost = 3
 
 /obj/effect/proc_holder/changeling/sting/mute/sting_action(mob/user, mob/living/carbon/target)
 	add_logs(user, target, "stung", "mute sting")
-	target.silent += 30
+	target.reagents.add_reagent("mutetoxin", 5)

```

Comment:
Setting silent directly will be better, IMO; means someone can't just pop pentetic or calomel and scream their head off.